adafruit_connection_manager.py

```diff
@@ -203,6 +204,18 @@ def _get_key_for_socket(self, socket):
         except StopIteration:
             return None
 
+    @property
+    def open_sockets(self) -> int:
+        """Get the count of open sockets"""
+        return len(self._open_sockets)
+
+    @property
+    def freeable_open_sockets(self) -> int:
+        """Get the count of freeable open sockets"""
+        return len(
+            [socket for socket, free in self._available_socket.items() if free is True]
+        )
+
```

Comment:
Does `freeable` mean `available` or `unused`? If that's the intention, I think that would be a better name.
As about `num_` or `_count` would be better, since you are returning a count.

As above, what is the use case of having this count available?

Also instead of creating a list and getting its length, I think you could do something like:
```py
return self._available_socket.values().count(True)
```
That brings up another question for me, not having read all the code. Is `._available_socket` really a list of all sockets. If not, why not just have a set that contains only free sockets, instead of a dict with key socket and value True when free?

Reply:
@dhalbert so this was how the original code in requests worked...

`_available_socket` is a list of sockets that are being tracked and whether or not it's available (but not closed)

I can update this to just a list of truely `available` and then update this method to `available_socket_count`

_I'll also update `open_sockets` to `open_socket_count`_

Does that work for you?